1.1/geo.ttl

```diff
@@ -523,6 +525,7 @@ geosparql:
 	rdfs:isDefinedBy geosparql: , geosparql-spec: ;
 	skos:definition """The Discrete Global Grid System (DGGS) serialization of a geometry"""@en ;	
 	skos:prefLabel "as DGGS"@en ;
+	skos:scopeNote "This property is likely to be used with specific DGGS' literal types, such as auspuxDggsLiteral, which are sub classes of the essentially abstract dggsLiteral" ;
```

Comment:
1. "sub classes" should be "subclasses"
2. "dggsLiteral" should be "DGGS literal" (because as a rule we use the skos:prefLabel in sentences.)

Reply:
Fixed as requested in commit https://github.com/opengeospatial/ogc-geosparql/pull/162/commits/e8fae58e396d5a1c9762cc55929e3ca0cfa17f9e